apps/spruce/src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -51,7 +52,11 @@ export const AuxiliaryDropdown: React.FC<AuxiliaryDropdownProps> = ({
     },
     {
       "data-cy": "auxiliary-dropdown-project-health",
-      text: "Project Health",
+      text: (
```

Comment:
<img width="221" alt="image" src="https://github.com/user-attachments/assets/e66b4bc8-87d2-4f5b-b14d-a58b1ca2114b" />
The text and badge should be side by side not stacked